### PR TITLE
Modernize project

### PR DESCRIPTION
**Pull request by gaborbernat**
Signed-off-by: Bernát Gábor <bgabor8@bloomberg.net>

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+build-backend = "hatchling.build"
+requires = ["hatchling>=1.11", "hatch-vcs>=0.2"]
+
+[project]
+name = "pytest-env"
+description = "py.test plugin that allows you to add environment variables."
+readme = "README.md"
+license.file = "LICENSE"
+maintainers = [{ name = "Bernát Gábor", email = "gaborjbernat@gmail.com" }]
+urls.Documentation = "https://bump-deps-index.readthedocs.io"
```
Comment (bfontaine) [inline pyproject.toml:11]: This seems unrelated.

Comment (gaborbernat) [reply pyproject.toml:11]: Yeah, leftover. Pr welcome.
